build.yaml

```diff
@@ -14,7 +14,7 @@ cassandra:
   - 2.1
   - 2.2
   - 3.0
-  - 3.10
+  - 3.7
```

Comment:
Also it wasn't testing against 3.10 despite at some point changing 3.7 -> 3.10, don't know why Job Creator kept deploying against 3.7